jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -29,7 +29,6 @@ fun <T> ComboBox<T>.populateValues(selected: T? = null, block: () -> List<T>) {
             model.removeAllElements()
             values.forEach { model.addElement(it) }
             this.selectedItem = selected
-            this.isEnabled = values.isNotEmpty()
```

Comment:
This is used in several places, is this going to affect other spots negatively?

Reply:
This affect iamRole and bucket selection in Lambda creation panel. Reworked the way for fixing the bug to avoid affecting the current behavior.